adapter/exporter/src/main/java/org/mobilitydata/gtfsvalidator/exporter/ProtobufNoticeExporter.java

```diff
@@ -964,10 +964,24 @@ public void export(final DecreasingStopTimeDistanceNotice toExport) throws IOExc
                 .setOtherCsvKeyName(String.valueOf(toExport.getNoticeSpecific(KEY_COMPOSITE_KEY_SECOND_VALUE)))
                 .setAltEntityValue(String.valueOf(toExport.getNoticeSpecific(KEY_STOP_TIME_STOP_SEQUENCE)))
                 .setValue(String.valueOf(toExport.getNoticeSpecific(KEY_STOP_TIME_SHAPE_DIST_TRAVELED)))
+                .setAltEntityName(String.valueOf(toExport.getNoticeSpecific(KEY_STOP_TIME_PREVIOUS_SHAPE_DIST_TRAVELED)))
+                .setParentEntityName(String.valueOf(toExport.getNoticeSpecific(KEY_STOP_TIME_PREVIOUS_STOP_SEQUENCE)))
+                .build()
+                .writeTo(streamGenerator.getStream());
+    }
+
+    @Override
+    public void export(final DecreasingShapeDistanceNotice toExport) throws IOException {
+        protoBuilder.clear()
+                .setCsvFileName(toExport.getFilename())
+                .setSeverity(GtfsValidationOutputProto.GtfsProblem.Severity.ERROR)
+                .setEntityId(toExport.getEntityId())
+                .setAltEntityValue(String.valueOf(toExport.getNoticeSpecific(KEY_SHAPE_PT_SEQUENCE)))
+                .setValue(String.valueOf(toExport.getNoticeSpecific(KEY_SHAPE_DIST_TRAVELED)))
                 .setAltEntityName(
-                        String.valueOf(toExport.getNoticeSpecific(KEY_STOP_TIME_PREVIOUS_SHAPE_DIST_TRAVELED)))
+                        String.valueOf(toExport.getNoticeSpecific(KEY_SHAPE_PREVIOUS_SHAPE_PT_SEQUENCE)))
```

Comment:
Indeed it is a wanted change. I suppose that GitHub flagged this as modification instead of new addition because `DecreasingShapeDistanceNotice` export process which is implemented just after `DecreasingStopTimeDistanceNotice` export process have similarities

Reply:
👍